### DISCUSSION TITLE
GitHub Page does not find my repository files

### DISCUSSION THREAD

**Discussion by D3vil0p3r** [community/community#23112] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everyone,<br>
I have a GitHub repository where I pushed some my Arch packages that I would like to use as a repository by using <code>pacman</code> on Arch Linux.<br>
My repository is: <a href="https://github.com/Athena-OS/athena-repository" rel="noopener nofollow ugc">https://github.com/Athena-OS/athena-repository</a></p>
<p>and I followed GitHub official documentation for creating my GitHub page: <a href="https://athena-os.github.io/athena-repository/" rel="noopener nofollow ugc">https://athena-os.github.io/athena-repository</a> and then on the Settings of the repository → Pages, I set “main” as Branch and /root as folder. And I saved this change.</p>
<p>The GitHub page on <a href="https://athena-os.github.io/athena-repository/" rel="noopener nofollow ugc">https://athena-os.github.io/athena-repository</a>  can be accessed correctly, but when I try to access to the files specified on my GitHub repo at <a href="https://github.com/Athena-OS/athena-repository" rel="noopener nofollow ugc">https://github.com/Athena-OS/athena-repository</a>, I get 404 error, for example if I try to access to <a href="https://athena-os.github.io/athena-repository/x86_64" rel="noopener nofollow ugc">https://athena-os.github.io/athena-repository/x86_64</a> or directly to a file <a href="https://athena-os.github.io/athena-repository/x86_64/archlinux-tweak-tool-1.0.0-1-x86_64.pkg.tar.zst" rel="noopener nofollow ugc">https://athena-os.github.io/athena-repository/x86_64/archlinux-tweak-tool-1.0.0-1-x86_64.pkg.tar.zst</a>.</p>
<p>How can I fix this issue?</p>

**Comment by yoannchaudet** [comment]
<p>Hello and welcome here!</p>
<p>The file you are looking for just does not exist in your repo:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/5/45fb891676096d54761460e57c2d33a54963f06d.png" data-download-href="/uploads/short-url/9Z5RbjuFFo37rWr745VO9BOIqjb.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181074909-53b1020a-e2d8-4f13-be86-9349c0cbea49.png" alt="image" data-base62-sha1="9Z5RbjuFFo37rWr745VO9BOIqjb" width="690" height="393" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/45fb891676096d54761460e57c2d33a54963f06d_2_690x393.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/5/45fb891676096d54761460e57c2d33a54963f06d_2_1035x589.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/5/45fb891676096d54761460e57c2d33a54963f06d_2_1380x786.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/5/45fb891676096d54761460e57c2d33a54963f06d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">2184×1246 290 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>Also GitHub Pages is not really meant for hosting binaries like you intend to do. We are mostly catering web pages. For that reason, we don’t enumerate folders which is why <a href="https://athena-os.github.io/athena-repository/x86_64">https://athena-os.github.io/athena-repository/x86_64</a> gives you a 404.</p>

**Comment by D3vil0p3r** [comment, marked as answer]
<p>Ok, I solved the issue despite it was very strange…</p>
<p>In practice, for first I created an index.md and .yaml files on my GitHub repo root where placing the index page of the website. Then, according the documentation, on the Settings of the repository → Pages, I set “main” as Branch and /root as folder. And I saved this change.</p>
<p>After this action, that should correspond to a Pull Request, an automatic job starts on GitHub that you can check on “Action” tab of your repository.</p>
<p>I noted that this action failed when Jekyll was building and the error was related to the presence on my repository of files as broken links. When I deleted these broken links (that were not called by my index page), the automatic task starts again and at the end it finished well. At this point I was able to retrieve my files by accessing to the GitHub page (e.g. <a href="https://athena-os.github.io/athena-repository/x86_64/athena-calamares-3.2.59-1-x86_64.pkg.tar.zst" rel="noopener nofollow ugc">https://athena-os.github.io/athena-repository/x86_64/athena-calamares-3.2.59-1-x86_64.pkg.tar.zst</a>)</p>
